### ISSUE TITLE
2.2.2.2 What is a clarification of a single answer?

### ISSUE THREAD

**Issue by 24RMiller** [gryphonshafer/Quizzing-Rule-Book#213]
Over the years I have seen many disagreements over whether the answer for an INT or MA is a clarification of a single answer or two separate answers. Here are just a few examples that were all appealed at Internationals 2024:

1. Q: Rather, he made himself nothing how?  A: By taking the very nature of a servant, being made in human likeness
2. Q: We continually ask God to fill you with the knowledge of his will through what? A: All the wisdom and understanding that the Spirit gives.
3. Q: Enslaved by what? A: Them (those weak and miserable forces)

The first two were MAs are were upheld as MAS. The third was an INT and upheld as an INT. My personal opinion is that all three are clarifications of one answer, but #1 has the best argument for being an MA. #1 is answered by 2 things, and could be argued either way whether being made in human likeness is a clarification of how he took the very nature of a servant or not. #2 seems to be one thing: "all the wisdom and understand that the Spirit gives." In my opinion, a valid ma would be "the Spirit gives what?" I view #3 the same as #2, as the valid MA there is "what forces?"

I've discussed these inconsistencies with several quizzers and officials, and we always struggle to come up with a definitive definition. One quizzer suggested that the clarification rule only applies to verbs, and not adjectives. According to him, the question: "Ryan is what?" with the answer: "running to the store and his house" is an int. However the question: "Ryan has what" with the answer: "brown eyes and hair" would be an MA. His argument is that the former would be a clarification of where you're running, while the latter is not a clarification of brown, but 2 brown things. 

I feel like this is an area where this is often not consistency, which makes it hard for quizzers to know what's an MA and what's not. Off the top of my head I can not think of any specific way to word this rule better to avoid this middle ground, but maybe I am just overblowing a problem that isn't there. I would be curious, first off for opinions about the examples given, and second off for general thoughts about the clarification rule.

**Comment by l-koral** [comment]
From how I see it, the difference between a ruling for an MA and an INT is the placing of words and their definitions. However I possibly have a different take on the rulings for these questions. And although I would like to explain what I would rule those questions as, I am certain that others would disagree with my thoughts. 
It has been this way for about as long as MAs and INTs have not been in existence: many individuals disagree on whether a question is one or the other. At times, a definitive answer has been reached as to the judgement of the ruling, with one side having a seemingly clearer understanding of the way the question is worded. Yet there are many instances, at Internationals especially, where a heated argument--among officials during the quiz, bystanders, or quizzers after the completion of the quiz--about the ruling. And that debate sometimes does not resolve itself. 
So, because of this, plenty of quizzers and coaches alike have advocated for the merging of Multiples and Interrogatives. Although I am not sending this message as a declaration that one side of the above argument is more valid than the other, I am wanting this: that quizzing be as enjoyable as possible, allowing those involved to glorify God. As far as I am concerned, there is no problem with a friendly disagreement about a ruling. Yet it is when a the conversation turns from amiable to almost hostile that I begin to have a problem. (I am not declaring this to be the case however.) Consequently, I would, in trying to help make us think, like to call this into attention. This may not be an issue as it common for me to think of every possible aspect of an issue and elaborate on those thoughts. Yet quizzing now has reached a point where it may have too many vague rules that are difficult to understand, even for a seasoned quizzer or coach. This may be more of an issue that is logistical in nature than it is just hardcore Bible Quizzing. If a quizzing rule is argued about because it is unclear or too wordy, then it should probably be reconsidered.
So I am, therefore, asking that a reconsideration of the problem between MAs and INTs be talked about. Maybe I just repeated what Ryan just said, but what I hope to show is that Bible Quizzing should examine its foundation to see where the building lays. Maybe Bible Quizzing is too complicated and maybe not, but whatever the case, I advise that special care be taken to ensure that the delicate program that has been instituted does not crumble under too heavy or burdensome of requirements.

**Comment by kclimenhaga** [comment]
The idea of combining INTs and MAs has been raised to me before, and my concern is that some MAs would not be valid as INTs. For example (obviously this is a CVRMA, not a straight MA, but it works as an illustration): 
Q: According to Luke 2:4, what town?
A: the town of Nazareth in Galilee; Bethlehem the town of David

These two towns are in the same verse, but they are in different places in the verse: "So Joseph also went up from **the town of Nazareth in Galilee** to Judea, to **Bethlehem the town of David**..." Because of this, if you don't indicate that it is a MA, the question is confusing and misleading and can't be used. 

Of course, there aren't that many questions like this, although it is a decent proportion of the MAs from what I recall. The other reason why I am in favour of keeping MAs as a separate question type is for the competition aspect. I think it is valuable for quizzers who are new at specialization to have a fairly small question type like MAs to be able to specialize in. Even though MAs aren't really easier than other question types, they can feel easier for new quizzers, so I think they are important in that way.

Regarding a clarification of the rule, it is a bit hard to make it more clear, because without a decent understanding of syntax, it can be hard to tell what is a MA and what isn't. I would explain it as "if both parts of the answer could answer that question, it is a MA". So for your examples:
1. This is a MA, because both of the answers (1-by taking the very nature of a servant, and 2-being made in human likeness) would make sense as answers to "he made himself nothing how?"
2. This is a MA, because both of the answers (1-the wisdom that the Spirit gives, and 2-the understanding that the Spirit gives) make sense as answers. I agree that this one is a bit harder to understand, because of the way it is worded, but there are two answers, which at their core are "wisdom" and "understanding". Those are the two things being linked by the "and", and so they are separate answers to the question.
3. This is an INT, because "weak" and "miserable" are not separate answers to "enslaved by what?" If the question were "what forces", then it would be an MA, but we are not enslaved by one set of weak forces and a different set of miserable forces, it is an INT with the answer "what forces". 

With your other example, "Ryan is what" (running to the store and his house) is an INT, but "Ryan is running where" is a MA. 

The other point I would make about clarifying this rule is that, most of the time, if a quizzer knows the material, whether a question is labelled as an INT or a MA does not make it easier or harder to answer the question. For example, in your example 2, if a quizzer thinks that "the wisdom and understanding that the Spirit gives" is one answer, then the quizzer will try to think of what the other answer is, but the quizmaster should have called the quizzer correct by then. In example 3, the quizzer isn't likely to say "those weak" and nothing else, and if they do, they should be called incorrect. The quizzer might misremember the verse and say "those weak forces", but "and miserable" is important enough information that it would be needed even apart from the INT/MA issue.

So in short, I think there are reasons to keep INTs and MAs. If there is a better way to word the rule, I'd love to hear it, but I can't come up with one myself.

**Comment by ARMediting** [comment]
MAs are something of a self-perpetuating problem, where separating them out makes them interesting to study and jump on, but having a limited amount of them means quizzers probably have a list of them, jump on one or two syllables, and are more likely caught off guard when a question isn't on their list. (This is the quizzers' own fault, yet the problem remains.)

**Comment by scrobiculatus** [comment]
> The idea of combining INTs and MAs has been raised to me before, and my concern is that some MAs would not be valid as INTs.

Is this the only problem with combining MAs and INTs? If so, would it be a problem to just... make the questions like this not be valid interrogatives?

**Comment by kclimenhaga** [comment]
I think it is also valuable to inform quizzers when a question has multiple parts, so that when they give the first part and aren't called correct, they understand that there is a second answer they need to give. It is definitely one solution to just get rid of non-contiguous MA questions, but I don't see the need to remove MAs either.
